### PR TITLE
All should be good and matching airdates now.

### PR DESCRIPTION
**Pull request by miigotu**

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: gui/slick/views/comingEpisodes.mako [hunk 1]
```diff
@@ -112,9 +112,8 @@
 
         <tr class="${show_div}">
             <td align="center" nowrap="nowrap">
-                <% airDate = sbdatetime.sbdatetime.sbfdatetime(cur_result['localtime']).decode(sickbeard.SYS_ENCODING) %>
-                <% isoDate = sbdatetime.sbdatetime.convert_to_setting(cur_result['localtime']).isoformat('T') %>
-                <time datetime="${isoDate}" class="date">${airDate}</time>
+                <% airDate = sbdatetime.sbdatetime.convert_to_setting(network_timezones.parse_date_time(cur_result['localtime'], cur_result['airs'], cur_result['network'])) %>
+                <time datetime="${airDate.isoformat('T')}" class="date">${sbdatetime.sbdatetime.sbfdate(airDate)}</time>
```
Comment (fernandog) [inline gui/slick/views/comingEpisodes.mako]: @miigotu @OmgImAlexis 
Its not showing time anymore. Also not respecting date-time setting user choose in SR settings.

**Comment by fernandog** [comment]
image

File: gui/slick/views/comingEpisodes.mako (hunk 1 above)
Comment (fernandog) [inline gui/slick/views/comingEpisodes.mako]: image

Comment (miigotu) [inline gui/slick/views/comingEpisodes.mako]: @OmgImAlexis fixed it. was using sbdate instead of sbfdatetime by accident.

**Pull request merged**